### PR TITLE
Remove PWA Starter Kit pointers

### PR DESCRIPTION
**Pull request by arthurevans**

### CODE REVIEW

**Comment by arthurevans** [comment]
Staged: https://20200309t230003-dot-polymer-library.appspot.com

Had a bunch of build issues, requiring me to update the gulpfile and bump some dependencies.

File: gulpfile.js [hunk 1]
```diff
@@ -266,16 +275,22 @@ gulp.task('js', 'Minify JS to dist/', ['jshint'], function() {
...
+gulp.task('js', js);
 
-gulp.task('build-bundles', 'Build element bundles', function() {
+const buildBundles = function() {
   return merge(
     $.run('polymer build').exec(),
     $.run('polymer build', {cwd: 'app/2.0/samples/homepage/contact-card'}).exec(),
     $.run('polymer build', {cwd: 'app/2.0/samples/homepage/google-map'}).exec())
-});
+  }
```
Comment (bicknellr) [inline gulpfile.js]:
```suggestion
};
```

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "postinstall": "bower install",
+    "oldpostinstall": "bower install",
```
Comment (bicknellr) [inline package.json]: If this isn't necessary anymore, we might as well remove it.

Comment (arthurevans) [reply package.json]: I thought we still needed the bower stuff for some samples, but on checking, I found it was no longer used. Yay!.

**Comment by arthurevans** [comment]
Thanks @bicknellr ... I updated some of the dependencies and it now seems to build on Node 12 & 13.

**Comment by arthurevans** [comment]
Thanks. Removed the references to the old samples--and bower_components, too.

**Pull request merged**